azure-pipelines.yml

```diff
@@ -42,6 +42,7 @@ jobs:
     envs:
       - macos: py36
         name: py36_test
+        posargs: -n=1
```

Comment:
Why this change?

Reply:
Race condition on the macos builds only.